src/aat/resources/features/F-1005 - GlobalSearch - Search cases/F-1005_CreateCasePreRequisiteCaseworker.td.json

```diff
@@ -0,0 +1,4 @@
+{
+	"_guid_": "F-1005_CreateCasePreRequisiteCaseworker",
+	"_extends_": "F-1004_CreateCasePreRequisiteCaseworker"
```

Comment:
Bit picky - but if referring to something in F-1004 from F-1005, should it be in a common location, that can then be referenced from F-1004 & F-1005 ?

Reply:
Hi Gareth, doesn't make sense to have a file for case creation for a specific case type under the common files, as it is used only for these 2 features. I can either leave it as it is or I can not extend and add all the same information to the F-1005 file.